### DISCUSSION TITLE
How to find the parameter responsible for the color of the Top File Menu Bar?

### DISCUSSION THREAD

**Discussion by Yulian-M** [nptr/msstyleEditor#140] [Q&A]
Hello! 👋
Thank you very much for developing such a cool program for customizing styles in Windows! ⚡️🌌🖥
It really became an unexpected problem for me when I switched from Windows 7 to 10, and was faced with the fact that the "Window Color and Appearance" tool was completely cut out...🔪
But I found your wonderful program and I believe that it will definitely help me! 💽

I need to change the appearance of only one detail, this is the Top Menu Bar (perhaps it is called differently, where it says "File, Edit, View...") in program windows that depend on the Windows theme.
Specifically, in my case, this is the Cockos Reaper program

You see, even when I set up a dark system theme, or any other extraneous one through the Personalization window, this menu remains white in Reaper, and in several other programs 🤷‍♂️.
![Menu bar with Black theme](https://github.com/nptr/msstyleEditor/assets/165702252/c2a3f293-f8b8-425d-9481-5107d4c6a0f2)

But when I turn on one of the themes with increased contrast, this line finally turns black. But I can’t imagine how you can work with such a topic for at least 10 minutes, it’s just painful for the eyes. Thus, I made sure that changing the color of this line by changing the system theme is real ✳️.
![Menu bar with High Contrast theme](https://github.com/nptr/msstyleEditor/assets/165702252/518be8ed-5f4c-4f45-a053-ece16f8ae7ee)

❓ I wanted to ask if you know which of the msstyle file options, such as aero, is responsible for this menu?

I tried changing almost all the options named Menu Bar or something like that, but I couldn't find the right one...

I will be grateful for any help!
And I am already very grateful to you for your open source software development! 🙌🔝

**Comment by TrantaLocked** [comment]
I really am not experienced with this but I made an attempt that works for a solid color fill.

The problem is I haven't figured out yet how to eliminate the white line at the bottom for themes not using UNKNOWN_5100_COLORLIST like Aerolite.

The parts are Menu -> BARBACKGROUND and BARITEM for text. For themes with the colorlist collection properties under BARBACKGROUND, change the colors in UNKNOWN_5100_COLORLIST without any further editing. For my theme that uses the Menu bar image, this is how I edited my BARBACKGROUND by changing BGTYPE from IMGFILE to BORDERFILL and adding FILLCOLOR to get it to look black which I verified using the test button but the margin numbers don't seem to matter:

![new](https://github.com/nptr/msstyleEditor/assets/18237916/3939586e-6a87-4665-aec1-081d6b199f6d)

Still trying to figure out how to eliminate that white line on the bottom that separates the Menu bar and the window contents. The Aerolite theme doesn't seem to have it but it has these weird Unknown 5100 and 5101 colorlist properties that the editor doesn't show an item to add. If your current theme already has the layout with the colorlist you might be able to just change the fill and line colors in one of the colorlist collections.

![FRFFR](https://github.com/nptr/msstyleEditor/assets/18237916/cf463848-3149-44a0-99ce-220064e97c98)

The line is most visible in apps where the content is dark like Adobe Premiere.

Update: I tried editing the image to all black in photoshop but the white line is still there so clearly there must be another necessary item to change it. 

After more testing I found the UNKNOWN_5100_COLORLIST colors 0 (enabled window) and 2 (disabled window) are related to the color of the top and bottom lines, and 1 and 3 for fill color, but that is only when messing with the Aerolite theme. I don't know how to manually add that property so I can modify the lines in my existing Aero 7 Shiny theme.

**Reply by Yulian-M** [reply]
Thank you very much for sharing your experience with msstyleEditor! 🙌
This is a really interesting and powerful tool that needs more detailed study.
I think I will come back to this issue and will definitely try your method! 📃
I hope you will succeed, or perhaps have already succeeded in removing this white line! It is strange, but it seems that it is missing on my screenshot. 🤔
Indeed, often you want to get the appearance of the window in accordance with your preferences! 🖼
It is a pity that Windows does not provide such an opportunity in the form of standard tools. ⚙️
Fortunately, there are developers of such software! 🙏
